static/app/views/performance/newTraceDetails/trace.tsx

```diff
@@ -213,15 +213,14 @@ function Trace({
           return;
         }
 
-        manager.onScrollEndOutOfBoundsCheck();
         setDetailNode(maybeNode.node);
         roving_dispatch({
           type: 'set index',
           index: maybeNode.index,
           node: maybeNode.node,
         });
 
-        manager.scrollRowIntoViewHorizontally(maybeNode.node);
+        manager.scrollRowIntoViewHorizontally(maybeNode.node, 0);
```

Comment:
this was causing the view to animate on load, which is unnecessary motion sickness for everyone